src/styles/transformers/rtl.js

```diff
@@ -0,0 +1,78 @@
+const reTranslate = /((^|\s)translate(3d|X)?\()(\-?[\d]+)/;
+const reSkew = /((^|\s)skew(x|y)?\()\s*(\-?[\d]+)(deg|rad|grad)(,\s*(\-?[\d]+)(deg|rad|grad))?/;
+
+/**
+ * This function ensures that `style` supports both ltr and rtl directions by
+ * checking `styleConstants` in `muiTheme` and replacing attribute keys if
+ * necessary.
+ */
+export default (muiTheme) => (style) => {
+  // Left to right is the default. No need to flip anything.
+  if (!muiTheme || !muiTheme.isRtl) return style;
```

Comment:
Yeah, technically! I left it like this for now for consistency with auto-prefix because the auto-prefix transformer is also technically checking `muiTheme` for on/off behavior. I have an idea on how we can get the best of both worlds:

Something like this:
```js
export default function(muiTheme) {
  if (!muiTheme || !muiTheme.isRtl) return null;

  return (style) => {
    // impl here...
  }
}
```
I have to test, but the compose function might ignore nulls. So that way the transformer can decide based on muiTheme whether or not it should be included in the transfomer list.